pages/linux/slurmstepd.md

```diff
@@ -0,0 +1,4 @@
+# slurmstepd
+
+> Slurm daemon for managing and monitoring individual job steps within a multi-step job.
+> More information: <https://slurm.schedmd.com/slurmstepd.html>.
```

Comment:
```suggestion
> Slurm daemon for managing and monitoring individual job steps within a multi-step job.
> It should not be invoked manually.
> More information: <https://slurm.schedmd.com/slurmstepd.html>.

- Start the daemon:

`slurmstepd`
```

Here we can follow a similar format to `osx` daemon pages i.e https://github.com/tldr-pages/tldr/blob/main/pages/osx/wifivelocityd.md.